oxli/functions.py

```diff
@@ -12,40 +12,40 @@
 import khmer.utils
 
 
-def estimate_optimal_with_N_and_M(N, M):
+def estimate_optimal_with_K_and_M(K, M):
     """
     Utility function for estimating optimal counting table args where N is the
     number of unique kmer and M is the allotted amount of memory
     """
-    Z = math.log(2)*(M/float(N))
-    intZ = int(Z)
-    if intZ == 0:
-        intZ = 1
-    H = int(M/intZ)
-    M = H*intZ
-    f2 = (1-math.exp(-N/float(H)))**intZ
+    N = math.log(2)*(M/float(K))
+    intN = int(N)
+    if intN == 0:
+        intN = 1
+    X = int(M/intN)
+    M = X*intN
+    f2 = (1-math.exp(-K/float(X)))**intN
```

Comment:
`make pep8` doesn't complain about this, but `make autopep8` change these lines. Which one should we consider, @mr-c?